tests/designs/uart2bus/Makefile

```diff
@@ -27,10 +27,10 @@ ifeq ($(SIM),aldec)
     GPI_EXTRA=vhpi
 endif
 ifeq ($(SIM),modelsim)
-    GPI_EXTRA=fli
```

Comment:
I'd expect the Makefile.questa to change to detech GPI_IMPL=fli (which it does not do at the moment erroneously) and keep the design Makefiles as they are.